docs/HowTo/Configure/Configuration-File.md

```diff
@@ -14,6 +14,21 @@ allow you to specify the address of the engine and manager APIs.
 
 {!global/Manager-Configuration-File.md!}
 
+### Override configuration options
+
+You can override any manager configuration file options by:
+
+- Specifying multiple configuration files on the command line.
+  If the same configuration options are specified across multiple files, the values in the file specified latest on the
+  command line are used.
```

Comment:
suggestion: ... multiple files, the values defined in the latest file are used (overriding the former file's values).